makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY = python-dependencies parts codes composed merge reason
```

Comment:
@putmantime I added a makefile. There's a different number of ways to do this.

As all of the steps in your pipeline generate a single output file, this goes well with `make`, since that's it's basic use case; you define a "target" which is the name of a file, and then you define the steps to make that and its dependencies.

The `.PHONY` targets are just convenience. For each of the `.PHONY` targets, I gave them two dependencies/prerequisites. For example:

```make
codes: parts data/output/owl_files/code_classes.owl
```

I actually didn't even need to add `parts` as a prerequisite. Having `data/output/owl_files/code_classes.owl` is enough, but I added in the `.PHONY` target of the previous step in the pipeline since that makes it easier to read the `makefile` and understand how the steps are connected to each other. Might be easier to explain this over voice; IDK how versed you are in `make`.